tests/acceptance/stepDefinitions/generalContext.js

```diff
@@ -103,6 +118,16 @@ Then('the following success/error message should be displayed on the webUI', fun
     .text.to.equal(message)
 })
 
+Then('a error message should be displayed on the webUI containing following text', function(
```

Comment:
duplicate of the function above
for discussion see also https://github.com/owncloud/web/pull/4538

Reply:
not actually, it checks if the message contains given text